sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1927,7 +1927,8 @@ object SQLConf {
   val STATE_STORE_SKIP_NULLS_FOR_STREAM_STREAM_JOINS =
   buildConf("spark.sql.streaming.stateStore.skipNullsForStreamStreamJoins.enabled")
     .internal()
-    .doc("When true, this config will skip null values in hash based stream-stream joins.")
+    .doc("When true, this config will skip null values in hash based stream-stream joins. " +
+      "The number of skipped null values will be shown as custom metric of stream join operator.")
```

Comment:
Could you paste an output of what this looks like in the PR description ? Thx

Reply:
Please let me know what output you are looking for? The output of this doc?

Reply:
Oh, if you're asking about the output of the metric, it is just as other metrics you can see in SQL UI.

Reply:
Yes correct. Ok cool - the metrics are not logged anywhere is it ?

Reply:
It will be in `StateOperatorProgress`'s `customMetrics` too, as  `customStatefulOperatorMetrics`. So streaming query listener can access it too besides UI.

Reply:
Do you have output for that ? or is it not easy to reproduce ?

Reply:
I don't have UI output now. I don't own the stream job. As the change follows existing metric framework so I think if the unit test makes sure metric is updated, it should be as expected in places.

Reply:
Cool sounds good. Thx